src/aggregator/aggregator/capture/aggregator.go

```diff
@@ -131,6 +132,26 @@ func (agg *aggregator) AddForwarded(
 	return nil
 }
 
+func (agg *aggregator) AddPassThrough(
+	metric aggregated.Metric,
+	metadata metadata.TimedMetadata,
+) error {
+	// Clone the metric and timed metadata to ensure it cannot be mutated externally.
+	metric = cloneTimedMetric(metric)
+	metadata = cloneTimedMetadata(metadata)
```

Comment:
What would potentially mutate this data?

Reply:
dunno, this is a test aggregator so not critical to perf, going to leave as is respecting the original author knew what was up.